crates/aggchain-proof-builder/src/lib.rs

```diff
@@ -288,7 +288,7 @@ impl<ContractsClient> AggchainProofBuilder<ContractsClient> {
         let mut fep_inputs = FepInputs {
             l1_head: l1_info_tree_leaf.inner.block_hash,
             claim_block_num: request.end_block as u32,
-            rollup_config_hash,
+            rollup_config_hash: op_succinct_config.rollup_config_hash,
```

Comment:
just a bit below we have this: 

https://github.com/agglayer/provers/blob/8d796cff66dde253fa44fb118ad88d9db3b58387/crates/aggchain-proof-builder/src/lib.rs#L302-L303

we put as prover inputs the `aggregation_vkey_hash` and `range_vkey_commitment` that we take from the crate dependency.

I think we can either:

1. verify that it matches what you fetch from the newly added rpc
2. not relying on the `op-succinct-elfs` anymore, and use only the ones fetched from the rpc

maybe 1st one is the safest for now